build-tools/automation/azure-pipelines.yml

```diff
@@ -0,0 +1,223 @@
+# Xamarin.Android Pipeline
+
+trigger:
+  - master
+  - d16*
```

Comment:
Is there a way to specify a single character wildcard here? I might accidentally name a branch `d16-2-backport-some-thing`.

Can it do a placeholder for a single character? `d16-_` or regex would be `d16\-\d`?

Reply:
`?` can be used as a single char wildcard- https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml#wildcards